network-api/networkapi/wagtailpages/models.py

```diff
@@ -556,13 +560,45 @@ class IndexPage(FoundationMetadataPageMixin, Page):
         FieldPanel('intro'),
     ]
 
+    def get_entries(self):
+        return self.get_children().live().order_by('-first_published_at')
+
     def get_context(self, request):
         context = super().get_context(request)
         context = set_main_site_nav_information(self, context, 'Homepage')
         context = get_page_tree_information(self, context)
-        context['entries'] = self.get_children().live().order_by('-first_published_at')
+        context['entries'] = self.get_entries()[0:1]
         return context
 
+    @route('entries')
+    def generate_entries_set_html(self, request, *args, **kwargs):
+        """
+        Get a set of entries, as rendered HTML
+        """
+        
+        page = 0
+        
+        if 'page' in request.GET:
+            page = int(request.GET['page'])
+
+        # Do we want to make this a CMS-controllable value?
+        page_size = 12
```

Comment:
this would be `page_size = self.page_size` as value in absence of a query argument